bot/convert_currency.go

```diff
@@ -63,12 +77,13 @@ func ConvertMessage(message string) string {
 
 	// check amount was provided
 	if arraySize > 3 {
-		// convert amount to float64
-		amount, err = strconv.ParseFloat(messageArray[3], 64)
+		// Parse amount with support for both comma and period
+		parsedAmount, err := parseAmount(messageArray[3])
 		if err != nil {
 			fmt.Println("Erro: a string fornecida não pode ser convertida em float64")
```

Comment:
Not related to this patch, but maybe we can convert the code to english @caio-imperial and use some type of translation to guarantee the message based on some env Variable